inst/assets/bibliography.bib

```diff
@@ -1908,3 +1908,286 @@ @article{yaverouglu2014revealing
+    pages = {2618-2625},
+    url = {https://academic.oup.com/bioinformatics/article/32/17/2618/2450751?login=true},
+    year = {2016}
+}
+
+
+@article{Mallick2022,
+    author = {Mallick H},
+    title = {Differential expression of single-cell RNA-seq data using Tweedie models},
+    journal = {Statistics in Medicine},
+    volume = {41},
+    number = {18},
+    pages = {3492-3510},
+    url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9288986/},
+    year = {2022}
+}
+
+
+@article{Ding2022,
+    author = {Ding DY},
+    title = {Cooperative learning for multiview analysis},
+    journal = {Proceedings of the National Academy of Sciences},
+    volume = {119},
+    number = {38},
+    pages = {e2202113119},
+    url = {https://www.pnas.org/doi/full/10.1073/pnas.2202113119},
+    year = {2022}
+}
+
+
+@article{Van2007,
+    author = {Van der Laan MJ},
+    title = {Super learner},
+    journal = {Statistical Applications in Genetics and Molecular Biology},
+    volume = {6},
+    number = {},
+    pages = {Article25},
+    url = {https://www.degruyter.com/document/doi/10.2202/1544-6115.1309/html},
+    year = {2007}
+}
+
+
+@article{Mallick2023,
+    author = {Mallick H},
+    title = {An integrated Bayesian framework for multi-omics prediction and classification},
+    journal = {Statistics in Medicine},
+    volume = {43},
+    number = {5},
+    pages = {983-1002},
+    url = {https://pubmed.ncbi.nlm.nih.gov/38146838/},
+    year = {2023}
+}
+
+
+@article{Lloyd-Price2019,
+    author = {Lloyd-Price J},
+    title = {Multi-omics of the gut microbial ecosystem in inflammatory bowel diseases},
+    journal = {Nature},
+    volume = {569},
+    number = {7758},
+    pages = {655-662},
+    url = {},
+    year = {2019}
+}
+
+
+@article{Subramanian2005,
+    author = {Subramanian A},
+    title = {Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles},
+    journal = {Proceedings of the National Academy of Sciences},
+    volume = {102},
+    number = {43},
+    pages = {15545-15550},
+    url = {https://www.pnas.org/doi/10.1073/pnas.0506580102?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed},
+    year = {2005}
+}
+
+
+@article{Kou2020,
+    author = {Kou Y},
+    title = {Microbe-set enrichment analysis facilitates functional interpretation of microbiome profiling data},
+    journal = {Scientific Reports},
+    volume = {10},
+    number = {1},
+    pages = {21466},
+    url = {https://www.nature.com/articles/s41598-020-78511-y},
+    year = {2020}
+}
+
+
+@article{Langfelder2008,
+    author = {Langfelder},
+    title = {WGCNA: an R package for weighted correlation network analysis},
+    journal = {BMC Bioinformatics},
+    volume = {9},
+    number = {},
+    pages = {559},
+    url = {https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-9-559},
+    year = {2008}
+}
+
+
+@article{Geistlinger2023,
+    author = {Geistlinger L},
```

Comment:
Authors should be written either as "First Last", or "Last, First" (note the comma in the latter option). 

Ideally, full name is written instead of the first letter.

-> Could you fix for this and the other references?